backends/dpdk/dpdkHelpers.cpp

```diff
@@ -765,6 +765,33 @@ bool ConvertStatementToDpdk::preorder(const IR::MethodCallStatement *s) {
             } else {
                 ::error("%1%: unhandled function", s);
             }
+        } else if (a->method->name == "mirror_packet") {
+            auto slot_id = a->expr->arguments->at(0)->expression;
```

Comment:
add test cases to verify
if the arguments count in extern function is validated in frontend